src/test/kotlin/tanvd/grazi/ide/language/MultiLanguageTextSupportTest.kt

```diff
@@ -0,0 +1,25 @@
+package tanvd.grazi.ide.language
+
+import tanvd.grazi.GraziConfig
+import tanvd.grazi.GraziTestBase
+import tanvd.grazi.ide.GraziLifecycle
+import tanvd.grazi.language.Lang
+
+
+class MultiLanguageTextSupportTest : GraziTestBase(true) {
```

Comment:
If it is actually `MultiLanguage text support` then you need to add English in text of test :)

Reply:
Add English text and now it's md file